nekoyume/Assets/_Scripts/UI/Widget/Craft.cs

```diff
@@ -359,6 +378,34 @@ private void CombinationConsumableAction(SubRecipeView.RecipeInfo recipeInfo)
             StartCoroutine(CoCombineNPCAnimation(consumable, requiredBlockIndex, true));
         }
 
+        public static Dictionary<int, int> CheckMaterial(Dictionary<int, int> materials)
```

Comment:
As far as I understand, the role of this method seems to be to check the original materials and inventory, and then return the missing materials.
But this method signature alone seems difficult to determine what role it works.
I sugest to change the name of parameter and name of method.

```suggestion
        public static Dictionary<int, int> GetMissingMaterials(Dictionary<int, int> originMaterialsOfRecipe)
```